### ISSUE TITLE
Gazelle Games tracker null category

### ISSUE THREAD

**Issue by criscross89** [Jackett/Jackett#15263]
### Have you checked our Troubleshooting page for your issue?

- [X] I have checked the Troubleshooting page

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Environment

```markdown
Using HTTP Client: HttpWebClient2
Using FlareSolverr: No
Using proxy: Disabled
App config/log directory: /config/Jackett
ThreadPool MaxThreads: 32767 workerThreads, 1000 completionPortThreads
Running in Docker: Yes (image build: unknown)
File /etc/issue: Arch Linux \r (\l)
Jackett variant: CoreLinuxAmdx64
OS version: Unix 6.1.82.0 (64bit OS) (64bit process)
Environment version: 6.0.28 (/usr/share/dotnet/shared/Microsoft.NETCore.App/6.0.28/)
Jackett Data will be stored in: /config/Jackett
Starting Jackett v0.21.2371
```

### Description

When searching GazelleGames tracker, see the null category in the results:
![image](https://github.com/Jackett/Jackett/assets/33426957/5973179f-2b49-4406-be24-a1008a534526)

### Logged Error Messages

```text
2024-04-21 10:39:59 	Info 	Torznab search in GazelleGames for takes two => Found 773 releases [1199ms]
2024-04-21 10:39:59 	Warn 	[gazellegames] Invalid Release: 'https://gazellegames.net/torrents.php?id=(removed)&torrentid=483538'. No categories provided.
[followed by more of the same]
Manual search in GazelleGames for it takes two => Found 4530 releases [4641ms]
```

**Comment by garfield69** [comment]
None of the current team have an account on GG, so we cant fix this without your help.
Please access the web site and do the search that generated the blank categories on Jackett.
Then dump the HTML with the instructions from https://github.com/Jackett/Jackett/wiki/Troubleshooting#how-to-provide-html-source

Remember to redact the file for privacy if you are posting the file here, alternatively email the file to 
 `garfieldsixtynine [at] gmail [dot] com` 
and I will update the indexer with the new categories.

**Comment by criscross89** [comment]
[Browse Torrents GazelleGames.txt](https://github.com/Jackett/Jackett/files/15056598/Browse.Torrents.GazelleGames.txt)

**Comment by garfield69** [comment]
Yeah, Nah. What you have provided is not the HTML I was looking for. Make sure to use the `save as HTML` option, even if you call the file whatever.txt
For it to be useful I need to see the HTML tags ;-) Thanks.

**Comment by garfield69** [comment]
also redact the auth and pass keys, these are meant to be confidential!

**Comment by ilike2burnthing** [comment]
They reported they're on v0.21.2371, and that commit was first available in v0.21.1491, so I don't think that's the issue.

Having a look at the HTML supplied in https://github.com/Jackett/Jackett/issues/15268, it seems that not all results have an `artistname`, in fact almost none of the results for `It Takes Two` seem to. While that's not the API, it might point towards the actual cause. It's then supposed to fallback to `CategoryID`, so possibly that's missing as well?

From a quick look at the `artistname` list, it seems that `Amiga CD32` and `Oculus Quest` need added as well.

**Comment by garfield69** [comment]
so on docker doesn't then check-for-updates replace the C# indexers?
How does docker Jackett report 0.21.2371 and yet clearly be using an old version for the gazellegames C# code?
We can tell this because the title for first entry in mynameisbogdan screen capture is slightly different from the 2nd title in criscross89 screen capture (as well as the category not showing of course).

**Comment by garfield69** [comment]
yes, but looking at the titles I compared, yours has update 1.0.2 at the end of the title while the OP has it in a different place, supporting your assertion that the OP docker image of Jackett is not using the same C# version of the gazelleGames indexer, and I do not understand how that can come about.

**Comment by garfield69** [comment]
Anyway, @criscross89 please update your docker Jackett image to the latest version using whatever the docker update command is, and see if that resolves your problem. Thanks.

**Comment by ilike2burnthing** [comment]
Oh wait! There's GazelleGames and then there's GazelleGames (API), with the users above using the former and Bogdan using the latter. The last time the GazelleGames indexer was individually updated was 3.5yrs ago, for an optional search setting, and before that it was over 5yrs ago.

So, before trying to fix that indexer, do we actually still need the HTML scraper?

**Comment by garfield69** [comment]
Oh well spotted. Completely missed that clue!
Given that Prowlarr only has the one GG indexer (which uses the API) I think we can dispose of our GG HTML scraper  indexer, since existing GG users should be able to use the API version.

**Comment by garfield69** [comment]
@criscross89  please ignore the instructions to update your docker Jackett image and instead switch to using the Gazelle Games (API) indexer.
We will be removing the old indexer as it is no longer supported ;-)
Thanks.

**Comment by ilike2burnthing** [comment]
@mynameisbogdan can we add `supportsFreeleechOnly` or `supportsFreeloadOnly` to the API indexer?

If not, can you add a freeleech filter setting?

**Comment by ilike2burnthing** [comment]
Ta!
